client/components/page/index.tsx

```diff
@@ -2,22 +2,39 @@
  * External dependencies
  */
 import { useEffect } from '@wordpress/element';
+import React from 'react';
```

Comment:
I'm forced to add it so TS can see `div`-tags, why?

Reply:
Imports have to be explicit in TS 🙂